### PR TITLE
Several updates

### PR DESCRIPTION
**Pull request by SoftCreatR**
- Updated Dockerfile to use Alpine 3.16
- Pinned package versions in Dockerfile
- Dockerfile & init script overhaul

### CODE REVIEW

**Comment by stnguyen90** [comment]
@SoftCreatR Thanks for the PR! 🤯 Please give us some time to review it.

**Comment by TorstenDittmann** [comment]
@SoftCreatR you need to bear with me a little bit 😄 these changes are not that easy to review. But this will happend during October - promise 👍🏻

**Comment by SoftCreatR** [comment]
If you could label this already as hf accepted, the timer can already start 😄

File: init-influxdb.sh [hunk 1]
```diff
@@ -4,138 +4,119 @@ set -e
 AUTH_ENABLED="$INFLUXDB_HTTP_AUTH_ENABLED"
 
 if [ -z "$AUTH_ENABLED" ]; then
-  AUTH_ENABLED="$(grep -iE '^\s*auth-enabled\s*=\s*true' /etc/influxdb/influxdb.conf | grep -io 'true' | cat)"
+	AUTH_ENABLED="$(grep -iE '^\s*auth-enabled\s*=\s*true' /etc/influxdb/influxdb.conf | grep -io 'true' | cat)"
```
Comment (stnguyen90) [inline init-influxdb.sh:7]: @SoftCreatR changing the tabs from 2 spaces to an actual tab adds a lot of whitespace changes that makes it harder to review this. Would you please change it back?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
I as able to build and run the branch on GitHub Codespaces:

image

File: init-influxdb.sh (hunk 1 above)
Comment (SoftCreatR) [reply init-influxdb.sh]: Done.

**Review by TorstenDittmann — APPROVED** [review]

**Review by stnguyen90 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**